app/models/rubygem.rb

```diff
@@ -163,7 +163,7 @@ def payload(version=versions.most_recent, host_with_port=Gemcutter::HOST)
       'gem_uri'           => "http://#{host_with_port}/gems/#{version.full_name}.gem",
       'homepage_uri'      => linkset.try(:home),
       'wiki_uri'          => linkset.try(:wiki),
-      'documentation_uri' => linkset.try(:docs),
+      'documentation_uri' => linkset.try(:docs) || version.documentation_path,
```

Comment:
Based on #967 and the code in `documentation_link` it looks like `linkset.try(:docs)` can be blank in which case `version.documentation_path` isn't called. Perhaps it just needs a `presence` check?

`'documentation_uri' => linkset.try(:docs).try(:presence) || version.documentation_path,`